core/templates/pages/oppia-root/routing/access-validation-backend-api.service.ts

```diff
@@ -36,6 +36,11 @@ export class AccessValidationBackendApiService {
   RELEASE_COORDINATOR_PAGE_ACCESS_VALIDATOR = (
     '/access_validation_handler/can_access_release_coordinator_page');
 
+  LEARNER_GROUP_EDITOR_PAGE_ACCESS_VALIDATOR = (
+    // eslint-disable-next-line max-len
+    '/access_validation_handler/can_access_edit_learner_group_page/<learner_group_id>'
```

Comment:
```suggestion
    '/access_validation_handler/can_access_edit_learner_group_page/' +
    '<learner_group_id>'
```

Generally, avoid using the pragma/bypass flag!

Reply:
Done.